content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -759,7 +760,7 @@ apply multiple fixes in between pausing and resuming without triggering unnecess
     REVISION  CHANGE-CAUSE
     1   <none>
     ```
-* Get the rollout status to ensure that the Deployment is updated successfully:
+* Get the rollout status to ensure that the existing ReplicaSet is not changed:
```

Comment:
```suggestion
* Get the rollout status to verify that the existing ReplicaSet has not changed:
```
?

Reply:
Thanks for the rewording, I've updated this sentence.